python/examples/core/1-first-steps.py

```diff
@@ -0,0 +1,264 @@
+"""
+.. _core-tutorial-first-steps:
+
+First steps with metatensor
+===========================
+
+This tutorial explores how data is stored inside metatensor's ``TensorMap``, and how to
+access the associated metadata.
+
+To this end, we will need some data in metatensor format, which for the sake of
+simplicity will be loaded from a file. The code used to generate this file can be found
+below:
+
+.. details:: Show the code used to generate the :file:`spherical-expansion.npz` file
+
+    ..
+
+        The data was generated with `rascaline`_, a package to compute atomistic
+        representations for machine learning applications.
+
+        .. _rascaline: https://luthaf.fr/rascaline/latest/index.html
+
+        .. literalinclude:: spherical-expansion.py.example
+            :language: python
+
+This file contains a machine learning representation (the spherical expansion) of all
+the atoms in a CO2 molecule. You don't need to know anything the spherical expansion to
```

Comment:
Maybe check if you can use `\rm` to typeset the elements non-italic.

```suggestion
the atoms in a :math:`CO_2` molecule. You don't need to know anything the spherical expansion to
```

This also applies to the CO2s in the tutorial.